### DISCUSSION TITLE
Smooth scroll support

### DISCUSSION THREAD

**Discussion by NatanFreeman** [hecrj/iced#1693] [Ideas]
When running the todo example I noticed that when scrolling, the items moved instantly instead of smoothly transitioning from their previous location to the new one like they would in a browser or many modern applications. I could not find I way of implementing this in the documentation. If I missed something please let me know.

Assuming smooth scroll is not implemented, will it be added? I understand implementing smooth scroll would be more complex and slightly more resource intensive, but I think the option is important for making smooth responsive feeling applications.

I am running the latest version on Windows 11 home. Maybe it's just a missing implementation. I would appreciate any feedback on this issue.

It appears this issue is the same as the one here #1674 .

**Comment by 13r0ck** [comment]
That could either be done with https://github.com/iced-rs/iced/pull/1647 or https://github.com/pop-os/cosmic-time
